### PR TITLE
Updated Calibration Manual

### PR DESCRIPTION
**Pull request by forReason**
Please Review!
I am not famillar with MD files but thought its probably just normal Markdown.
and again:
Please Review!

### CODE REVIEW

**Comment by lordofhyphens** [comment]
Sorry this took so long. Please read my comments and address.

**Review by lordofhyphens — CHANGES_REQUESTED** [review]

**Comment by lordofhyphens** [comment]
@forReason I left you feedback on what needed to change.

File: src/first-print/calibration.md [hunk 1]
```diff
@@ -34,3 +44,195 @@ print. Too little will result in gaps and poor inter-layer adhesion.
...
+- Try to measure the smallest number of layers possible from top as the slightest vertical misalignment of your layers will increase the measured value. Ideally a single layer would be best but very difficult with regular calipers.
+- Make sure you dont apply to much pressure with your caliphers (Make sure the measuring device is not biting into the filament during measurement.)
+
+
+**Do not forget to save Your calibration settings.**
+
+### Advanced tuning
+#### Requirements
+- You are able to print in reasonable quality after your "Basic Tuning"
+- Switch to expert mode
```
Comment (lordofhyphens) [inline src/first-print/calibration.md]: Expert mode is no longer a thing, so this can be removed.

**Comment by lordofhyphens** [comment]
@forReason any updates?

File: src/first-print/calibration.md (hunk 1 above)
Comment (forReason) [reply src/first-print/calibration.md]: what does that mean exactly? I haven't been in the scene for a wile. does it mean you only have the "simple" settings or you only have the "expert" settings?

**Comment by forReason** [comment]
Hello @lordofhyphens , I havn't been in the scene for awhile. I will review the changes this week

File: src/first-print/calibration.md (hunk 1 above)
Comment (lordofhyphens) [reply src/first-print/calibration.md]: We don't have simple or expert mode. We are all experts :D

**Comment by lordofhyphens** [comment]
@forReason Thank you.

**Comment by alranel** [comment]
Hi @forReason, any updates on this?

**Pull request closed without merging**